elastic/kibana
# [Fleet] Fix fetching package detail info after package policy creation

## Summary

Resolve https://github.com/elastic/kibana/issues/168942

Fix fetching package details info after installing the package during package policy creation,

It seems the cache for package details is sometime not correct just after installing a package during a package policy for example that PR fix that by forcing a refetch on mount.


## Test instructions

From a fresh install, setup fleet server, 
* try to add an integration with the multi step flow
* Try to click on view asset at the end of the flow it should redirect you correcty.
